.bazelignore

```diff
@@ -0,0 +1,24 @@
+node_modules
+packages/amp/node_modules
+packages/angular/node_modules
+packages/app-context/node_modules
+packages/cli/node_modules
+packages/codegen/node_modules
+packages/commerce-plugin-tools/node_modules
+packages/core/node_modules
+packages/email/node_modules
+packages/fiddle/node_modules
+packages/figma/node_modules
+packages/gatsby/node_modules
+packages/json-schema/node_modules
+packages/jsx/node_modules
+packages/native/node_modules
+packages/plugin-loader/node_modules
+packages/react/node_modules
+packages/shopify/node_modules
+packages/storybook/node_modules
+packages/utils/node_modules
+packages/vue/node_modules
+packages/webcomponents/node_modules
+packages/widgets/node_modules
+packages/wordpress/node_modules
```

Comment:
dang, I take it bazelignore doesn't take wildcards or any other type of pattern to block out all node_modules like we can in gitignore/npmignore?

Reply:
Doesn't look like it https://github.com/bazelbuild/bazel/issues/7093

Reply:
unfortunately does not. :-(